### ISSUE TITLE
time series plotting method

### ISSUE THREAD

**Issue by knaaptime** [spatialucr/geosnap#11]
it would be really useful to have a `.tsplot()` method on the `Dataset` class that arranges plots for each available time period on the same figure

**Comment by knaaptime** [comment]
I've been playing with `proplot`-based solution that looks really nice and is easy to format for a variable number of years


as an example:

```
f, axs = plot.subplots(ncols=len(dc.gdf.year.unique()))

for i, year in enumerate(dc.gdf.year.unique()):

    dc.gdf[dc.gdf.year==1990].plot(column='median_household_income', ax=axs[i], scheme='quantiles', k=8, legend=True, legend_kwds={'ncols':1, "loc":"b"})
    axs[i].format(title=year)

axs.format(suptitle='Washington DC Median Income')
axs.axis('off')
```

![image](https://user-images.githubusercontent.com/4213368/69987370-4034cf80-14f4-11ea-85ad-461a6f15f34d.png)

**Comment by knaaptime** [comment]
resolved by #248
